src/sql/src/parsers/create_parser.rs

```diff
@@ -977,6 +1064,29 @@ ENGINE=mito";
 
         let result4 = ParserContext::create_with_dialect(sql4, &GenericDialect {});
         assert!(result4.is_err());
+
+        let sql = r"
+CREATE TABLE monitor (
+  host_id    INT,
+  idc        STRING,
+  ts         TIMESTAMP TIME INDEX DEFAULT CURRENT_TIMESTAMP,
+  cpu        DOUBLE DEFAULT 0,
+  memory     DOUBLE,
+  TIME INDEX (ts),
+  PRIMARY KEY (host),
+)
+ENGINE=mito";
+
+        let result = ParserContext::create_with_dialect(sql, &GenericDialect {}).unwrap();
+
+        if let Statement::CreateTable(c) = &result[0] {
+            let ts = c.columns[2].clone();
+            assert_eq!(ts.name.to_string(), "ts");
+            assert!(matches!(ts.options[0].option, ColumnOption::Default(..)));
+            assert_eq!(ts.options[1].option, NotNull);
```

Comment:
assert `TIME INDEX` as well?

Reply:
The `time index` option is removed from options, but it should be in table constraints, I'll add an assertion.